gap/PGMatrix.gd

```diff
@@ -0,0 +1,96 @@
+#!
+#!
+#! @Chapter Hyperbolic lattice symmetries
+
+#!
+#! @Section Point group matrices
```

Comment:
```suggestion
#! @Section Point-group matrices
```

Reply:
Is there a technical/mathematical term for these matrices which describe the action of point-group elements on momenta?